pkg/hub/hub.go

```diff
@@ -69,6 +69,7 @@ func NewHub(L hclog.Logger, client *control.Client, feToken string) (*Hub, error
 	cfg.Logger = L.StandardLogger(&hclog.StandardLoggerOptions{
 		InferLevels: true,
 	})
+	cfg.StreamCloseTimeout = 60 * time.Second
```

Comment:
Purposely made this longer since I think servers can handle a bit more latency with bad clients.